src/erlcloud_aws.erl

```diff
@@ -967,3 +989,156 @@ error_msg( Message ) ->
 error_msg( Format, Values ) ->
     Error = iolist_to_binary( io_lib:format( Format, Values ) ),
     throw( {error, Error} ).
+
+
+
+%%====================================================================
+%% Test Functions
+%%====================================================================
```

Comment:
This is a good point, I think. This seems to be a pretty established convention in the repo.